plugins/repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureSnapshotRestoreTests.java

```diff
@@ -106,9 +106,9 @@ public void testSimpleWorkflow() {
         logger.info("-->  creating azure repository with path [{}]", getRepositoryPath());
         PutRepositoryResponse putRepositoryResponse = client.admin().cluster().preparePutRepository("test-repo")
                 .setType("azure").setSettings(Settings.settingsBuilder()
-                        .put(Repository.CONTAINER, getContainerName())
-                        .put(Repository.BASE_PATH, getRepositoryPath())
-                        .put(Repository.CHUNK_SIZE, randomIntBetween(1000, 10000), ByteSizeUnit.BYTES)
+                        .put(Repository.CONTAINER_SETTING, getContainerName())
```

Comment:
It should be. I'm puzzled why this didn't fail in my local verification build. Maybe because it's a third party test case? I'll correct the key anyway.